.github/workflows/release.yml

```diff
@@ -118,10 +118,15 @@ jobs:
           distribution: temurin
           java-version: 17
 
+      - name: Set up gradle
+        uses: gradle/actions/setup-gradle@v4
+        env:
+          SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
+          SONATYPE_KEY: ${{ secrets.SONATYPE_KEY }}
+          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+          GPG_PASSWORD: ${{ secrets.GPG_PASSWORD }}
```

Comment:
When splitting the gradle action in two, I wasn't sure which phase needs the environment vars, or if/how they get cached (by the gradle daemon?), so I just duplicated it to both the gradle setup and the gradlew steps. If this is unnecessary and stupid, please let me know!

Reply:
Only the gradlew step needs them, you can remove them from setup-gradle step. At least that is how we have them in the instrumentation repo.